src/boot/py/pyffi.ml

```diff
@@ -0,0 +1,68 @@
+
+(*
+   Miking is licensed under the MIT license.
+   Copyright (C) David Broman. See file LICENSE.txt
+ *)
+
+
+open Ast
+open Pyast
+open Pypprint
+open Ustring.Op
+open Msg
+
+let mk_py fi p = TmConst(fi, CPy(p))
+
+let externals =
+  let f p = mk_py NoInfo p in
+  [("pycall", f(Pycall(None,None))); ("pyimport", f(Pyimport))]
+
+let arity = function
+  | PyObject(_) -> 0
+  | Pyimport -> 1
+  | Pycall(None, None) -> 3 | Pycall(Some(_),None) -> 2 | Pycall(_,Some(_)) -> 1
+
+let rec val_to_python fi = function
+  | TmConst(_,CBool(v)) -> Py.Bool.of_bool v
+  | TmConst(_,CInt(v)) -> Py.Int.of_int v
+  | TmConst(_,CFloat(v)) -> Py.Float.of_float v
+  | TmConst(_,CPy(PyObject(v))) -> v
+  | TmSeq(_,s) ->
+    begin
+      try tmseq2ustring fi s |> Ustring.to_utf8 |> Py.String.of_string
+      with _ -> Mseq.to_list s |> Py.List.of_list_map (val_to_python fi)
+    end
+  (* | TmRecord(_,r1) ->
```

Comment:
I think you can just remove these two. The alternative might be to create dicts (or whatever python calls them, I forget), but maybe we add that once we have a use for it?

Reply:
Removed for now.